x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -1484,4 +1484,48 @@ export const tasks: TelemetryTask[] = [
       };
     },
   },
+  {
+    name: 'top_traces',
+    executor: async ({ indices, telemetryClient }) => {
+      const response = await telemetryClient.search({
+        index: indices.transaction,
```

Comment:
Do we need `indices.span` here as well?

Reply:
Yes, good point. And perhaps even errors?

Reply:
I don't know, do we?

Reply:
Fixed here https://github.com/elastic/kibana/pull/166263/commits/21ca0e6a43bf7ca36e1430157fac28fda08fb78d